src/widgets/splits/DraggedSplit.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "widgets/splits/Split.hpp"
+
+namespace chatterino {
+
+// Returns true if the user is currently dragging a split in this Chatterino instance
+// We need to keep track of this to ensure splits from other Chatterino instances aren't treated as memory we own
+[[nodiscard]] bool isDraggingSplit();
+
+// Set the current split dragging state
+// Controlled by Split's drag function
+void setDraggingSplit(bool isDraggingSplit);
```

Comment:
Look into the naming of this function

Reply:
I split this up into two functions instead, it was the clearest alternative I could find